addons/mail/static/src/models/messaging_menu.js

```diff
@@ -231,5 +231,13 @@ registerModel({
             inverse: 'messagingMenuAsPinnedAndUnreadChannel',
             readonly: true,
         }),
+        /**
+         * global JS generated ID for this record view. Useful to provide a
+         * custom class to autocomplete input, so that click in an autocomplete
+         * item is not considered as a click away from messaging menu in mobile.
+         */
+        viewId: attr({
+            default: _.uniqueId('o_messagingMenu_'),
```

Comment:
must be a compute, default is static

maybe just move in compute of custom class?

Reply:
Changes done